pymobiledevice3/lockdown.py

```diff
@@ -103,7 +101,7 @@ class LockdownClient(object):
     def __init__(self, serial: str = None, hostname: str = None, client_name: str = DEFAULT_CLIENT_NAME,
                  autopair: bool = True, usbmux_connection_type: str = None, pair_timeout: int = None,
                  local_hostname: str = None, pair_record: Mapping = None,
-                 pairing_records_cache_folder: Path = HOMEFOLDER):
+                 pairing_records_cache_folder: Path = get_home_folder()):
```

Comment:
`get_home_folder` will be called on each import, you better put the default `None` and call it inside the function.
Also, remove the mkdir from line 546/544

Reply:
done